tests/conftest.py

```diff
@@ -177,14 +180,14 @@ def describe_instances__one_instance(self):
                             "PrivateIpAddress": "172.31.51.34",
                             "ProductCodes": [],
                             "PublicDnsName": self.instance_public_dns,
-                            "PublicIpAddress": "3.235.31.42",
```

Comment:
Is this a problem that this number has changed by referencing instance_public_ip instead of hardcoded here, or does it not really matter the IP?

Reply:
No problem.  As I am sure you can tell, these are just captured responses from real calls to AWS.  Prior, we were not using the IP address so it was ignored. Now we are, so there are a few places in which those values need to be consistent.  I could have used the "3.x.x.x" address instead, but the "34.x.x.x" was the one that the instance was assigned when I was working on this PR.